package.json

```diff
@@ -30,6 +30,7 @@
     "broccoli-asset-rev": "2.4.6",
     "broccoli-clean-css": "1.1.0",
     "broccoli-concat": "3.0.2",
+    "broccoli-funnel": "^1.0.7",
```

Comment:
Does this dependency need to be added here or is it a sub-dependency of the `ghost-editor` package?